scripts/diff_to_added_lines.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import sys
+import unidiff
+import os.path
+import json
+
+if len(sys.argv) != 3:
+  print >>sys.stderr, "Usage: diff_to_added_lines.py diff.patch repository_root_directory"
+  sys.exit(1)
+
+diff_file = sys.argv[1]
```

Comment:
Ideally use `argparse` module rather than manually parsing arguments.

Reply:
Given these are fixed-position and fixed in number, I don't see it gains us anything in this case

Reply:
Fair enough.